pandas/core/tools/numeric.py

```diff
@@ -48,7 +48,7 @@ def to_numeric(arg, errors="raise", downcast=None):
         - If 'raise', then invalid parsing will raise an exception.
         - If 'coerce', then invalid parsing will be set as NaN.
         - If 'ignore', then invalid parsing will return the input.
-    downcast : {'integer', 'signed', 'unsigned', 'float'}, default None
+    downcast : {'int', 'signed', 'unsigned', 'float'}, default None
```

Comment:
needs to be a fix similar to what you did in pandas/core/arrays/sparse/array.py, this won't work

Reply:
Okay you mean something like this -
```
downcast : str
           Can be 'integer' or 'signed' or 'unsigned' or 'float', default is 'None'.
```

Reply:
I'd suggest
```
downcast : str, default None
    Can be 'integer', 'signed', 'unsigned', or 'float'.
```